libnavigation-ui/src/main/assets/sdk_versions/com.mapbox.libnavigation.ui

```diff
@@ -0,0 +1,2 @@
+libnavigation-ui/0.43.0-SNAPSHOT
```

Comment:
This can be removed for now (we're not including it in `libnavigation-core` either).